src/quill-editor.component.ts

```diff
@@ -97,6 +103,19 @@ export class QuillEditorComponent implements AfterViewInit, ControlValueAccessor
   onModelChange: Function = () => {};
   onModelTouched: Function = () => {};
 
+  @Input()
+  valueGetter: Function = (quillEditor: any, editorElement: HTMLElement) => {
```

Comment:
no need to type the functions as Function the fat arrow gives them the type implicit.

But you should type the return value of the functions ;)

Reply:
sure, thought it will not hurt anyone :) will change it.